code/ATMOSPHERICS/components/binary_devices/pump.dm

```diff
@@ -121,7 +121,7 @@ Thus, the two variables affect pump operation are set in New():
 	return 1
 
 //Radio remote control
-/obj/machinery/atmospherics/binary/pump/proc/set_frequency(new_frequency)
+/obj/machinery/atmospherics/binary/pump/set_frequency(new_frequency)
 	SSradio.remove_object(src, frequency)
 	frequency = new_frequency
 	if(frequency)
```

Comment:
If the child proc is the same on all of these, why not just define it at parent level?

Reply:
That is a very good point, one that 9 month ago me didn't realize. I will fix it.

Reply:
Addressed in the latest commit.